### ISSUE TITLE
[ERROR] app.firestore is not a function

### ISSUE THREAD

**Issue by fjfo** [Tiledesk/tiledesk-dashboard#42]
After: $ng server
** NG Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
Date: 2019-02-21T07:37:03.509Z
Hash: f2153a0bd2defb23b856
Time: 17905ms
chunk {inline} inline.bundle.js (inline) 5.79 kB [entry] [rendered]
chunk {main} main.bundle.js (main) 5.74 MB [initial] [rendered]
chunk {polyfills} polyfills.bundle.js (polyfills) 1.1 MB [initial] [rendered]
chunk {scripts} scripts.bundle.js (scripts) 720 kB [initial] [rendered]
chunk {styles} styles.bundle.js (styles) 973 kB [initial] [rendered]
chunk {upload.module} upload.module.chunk.js () 589 kB  [rendered]
chunk {vendor} vendor.bundle.js (vendor) 28.8 MB [initial] [rendered]
webpack: Compiled successfully

When I try to : http://localhost:4200/, I got nothing and in console show up with:

`************** APPMODULE ******************
AppComponent_Host.ngfactory.js? [sm]:1 ERROR TypeError: app.firestore is not a function
    at eval (firestore.js:35)
    at ZoneDelegate.invoke (zone.js:365)
    at Zone.run (zone.js:125)
    at NgZone.runOutsideAngular (core.js:4708)
    at new AngularFirestore (firestore.js:33)
    at _createClass (core.js:10943)
    at _createProviderInstance$1 (core.js:10907)
    at resolveNgModuleDep (core.js:10892)
    at _createClass (core.js:10941)
    at _createProviderInstance$1 (core.js:10907)
    at resolveNgModuleDep (core.js:10892)
    at NgModuleRef_.get (core.js:12129)
    at resolveDep (core.js:12619)
    at createClass (core.js:12489)
    at createDirectiveInstance (core.js:12326)
View_AppComponent_Host_0 @ AppComponent_Host.ngfactory.js? [sm]:1
proxyClass @ compiler.js:14659
DebugContext_.logError @ core.js:15038
ErrorHandler.handleError @ core.js:1510
(anonymous) @ core.js:5693
ZoneDelegate.invoke @ zone.js:365
Zone.run @ zone.js:125
NgZone.runOutsideAngular @ core.js:4708
(anonymous) @ core.js:5693
ZoneDelegate.invoke @ zone.js:365
onInvoke @ core.js:4760
ZoneDelegate.invoke @ zone.js:364
Zone.run @ zone.js:125
(anonymous) @ zone.js:760
ZoneDelegate.invokeTask @ zone.js:398
onInvokeTask @ core.js:4751
ZoneDelegate.invokeTask @ zone.js:397
Zone.runTask @ zone.js:165
drainMicroTaskQueue @ zone.js:593
Promise.then (async)
scheduleQueueDrain @ zone.js:552
scheduleMicroTask @ zone.js:560
ZoneDelegate.scheduleTask @ zone.js:387
Zone.scheduleTask @ zone.js:209
Zone.scheduleMicroTask @ zone.js:229
scheduleResolveOrReject @ zone.js:758
ZoneAwarePromise.then @ zone.js:847
PlatformRef.bootstrapModule @ core.js:5579
(anonymous) @ main.ts:11
./src/main.ts @ main.bundle.js:1760
__webpack_require__ @ inline.bundle.js:55
0 @ main.bundle.js:1768
__webpack_require__ @ inline.bundle.js:55
webpackJsonpCallback @ inline.bundle.js:26
(anonymous) @ main.bundle.js:1
AppComponent_Host.ngfactory.js? [sm]:1 ERROR CONTEXT DebugContext_ {view: {…}, nodeIndex: 1, nodeDef: {…}, elDef: {…}, elView: {…}}`

I took me few days with this bug. Any idea ?

**Comment by marcellokabora** [comment]
Yes like usual I got this error after I update to the last NPM, now we have to wait until someone of the community tells us how to fix it. Delete the node_modules and install all again the cash did not work this time :(

**Comment by andrealeo83** [comment]
Please checkout last commit https://github.com/Tiledesk/tiledesk-dashboard/commit/b209508b1cf2d8c535bc1d2f4ffc3b066392a610
